anta/models.py

```diff
@@ -71,11 +101,37 @@ class AntaTemplate(BaseModel):
 
     """
 
-    template: str
-    version: Literal[1, "latest"] = "latest"
-    revision: Revision | None = None
-    ofmt: Literal["json", "text"] = "json"
-    use_cache: bool = True
+    # pylint: disable=too-few-public-methods
+
+    __metaclass__ = SingletonArgs
```

Comment:
should probably be `class AntaTemplate(metaclass=SingletonArgs)`

Reply:
so yes it SHOULD (as `__metadata__` does not work anymore)
but also it is useless so it was
1. fixed in [350546d](https://github.com/arista-netdevops-community/anta/pull/654/commits/350546dfe6875c7aea77682e8fb4100ca514b719) _if we ever need this later_
2. removed in [96dc467](https://github.com/arista-netdevops-community/anta/pull/654/commits/96dc467a5341956cf4f881f770db1a9d685112f9)